install/kubernetes/cilium/values.yaml

```diff
@@ -908,6 +908,9 @@ resourceQuotas:
       # 15 "clusterwide" Cilium Operator pods for HA
       pods: "15"
 
+# skipSocketLB enables skipping the socket lb when running in kube-proxy free mode.
+skipSocketLB: false
```

Comment:
Is there a reason why this version of the flag doesn't mention the pod NS?

Reply:
Changed to the full name.